spec/support/devise.rb

```diff
@@ -0,0 +1,6 @@
+RSpec.configure do |config|
+  config.include Devise::Test::ControllerHelpers, type: :controller
+  config.include Devise::Test::ControllerHelpers, type: :view
+  config.include Devise::Test::IntegrationHelpers, type: :request
+  config.include Devise::Test::IntegrationHelpers, type: :feature
```

Comment:
This provides the [helper methods](https://github.com/heartcombo/devise#test-helpers) that make working with devise and rspec so much easier.